server/routes/payment.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const {
+  createPaymentMethod,
+  getPaymentMethods,
+  createCheckoutSession,
+} = require("../controllers/payment");
+const { authenticate } = require("../middleware/authenticate");
+
+const Router = express.Router();
+
+Router.post("/methods", authenticate(), createPaymentMethod);
+Router.get("/methods/:profile_id", authenticate(), getPaymentMethods);
+//might not need checkout
```

Comment:
Do you think we'll need the checkout methods?

Reply:
I agree it was more a learning experience with how Stripe worked.